### PR TITLE
docs: clean up inconsistent copyright notices

### PR DESCRIPTION
**Pull request by infinitewarp**
Unless otherwise stated, all source code files in this project are copyright Red Hat, Inc. and are licensed according to the terms of the LICENSE file.

### CODE REVIEW

File: quipucords/manage.py [hunk 1]
```diff
@@ -1,14 +1,3 @@
-#!/usr/bin/env python
```
Comment (bruno-fs) [inline quipucords/manage.py:1]: @infinitewarp Could this be the culprit causing issues? We might need this shebang line :thinking:

Comment (infinitewarp) [reply quipucords/manage.py:1]: Deleting the shebang here was a mistake. Thanks for catching it!

**Review by bruno-fs — APPROVED** [review]
lgtm

**Comment by abellotti** [comment]
Curious, because this code would live on a customer system (vs saas), do the files need to include something saying "for copyright notice see the LICENSE file ..." ?

**Comment by infinitewarp** [comment]
> Curious, because this code would live on a customer system (vs saas), do the files need to include something saying "for copyright notice see the LICENSE file ..." ?

The customer would have to try pretty hard to find any of the copyright/license-related comments since it's all bundled inside the container image(s).

That being said, the popular libraries that we use don't do include header notices everywhere. I would look to [Django](https://github.com/django/django) and [Django Rest Framework](https://github.com/encode/django-rest-framework) as leading examples. They have a LICENSE file at the top, and the rest of the code files are clean.

**Review by abellotti — APPROVED** [review]

**Pull request merged**